src/renderer/api/__tests__/kube-api-parse.test.ts

```diff
@@ -1,134 +1,115 @@
-    },
-  },
+  ["/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions/prometheuses.monitoring.coreos.com", {
+    apiBase: "/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions",
+    apiPrefix: "/apis",
+    apiGroup: "apiextensions.k8s.io",
+    apiVersion: "v1beta1",
+    apiVersionWithGroup: "apiextensions.k8s.io/v1beta1",
+    namespace: undefined,
+    resource: "customresourcedefinitions",
+    name: "prometheuses.monitoring.coreos.com"
+  }],
+  ["/api/v1/namespaces/kube-system/pods/coredns-6955765f44-v8p27", {
+    apiBase: "/api/v1/pods",
+    apiPrefix: "/api",
+    apiGroup: "",
+    apiVersion: "v1",
+    apiVersionWithGroup: "v1",
+    namespace: "kube-system",
+    resource: "pods",
+    name: "coredns-6955765f44-v8p27"
+  }],
+  ["/apis/stable.example.com/foo1/crontabs", {
+    apiBase: "/apis/stable.example.com/foo1/crontabs",
+    apiPrefix: "/apis",
+    apiGroup: "stable.example.com",
+    apiVersion: "foo1",
+    apiVersionWithGroup: "stable.example.com/foo1",
+    resource: "crontabs",
+    name: undefined,
+    namespace: undefined,
+  }],
+  ["/apis/cluster.k8s.io/v1alpha1/clusters", {
+    apiBase: "/apis/cluster.k8s.io/v1alpha1/clusters",
+    apiPrefix: "/apis",
+    apiGroup: "cluster.k8s.io",
+    apiVersion: "v1alpha1",
+    apiVersionWithGroup: "cluster.k8s.io/v1alpha1",
+    resource: "clusters",
+    name: undefined,
+    namespace: undefined,
+  }],
+  ["/api/v1/namespaces", {
+    apiBase: "/api/v1/namespaces",
+    apiPrefix: "/api",
+    apiGroup: "",
+    apiVersion: "v1",
+    apiVersionWithGroup: "v1",
+    resource: "namespaces",
+    name: undefined,
+    namespace: undefined,
+  }],
+  ["/api/v1/secrets", {
+    apiBase: "/api/v1/secrets",
+    apiPrefix: "/api",
+    apiGroup: "",
+    apiVersion: "v1",
+    apiVersionWithGroup: "v1",
+    resource: "secrets",
+    name: undefined,
+    namespace: undefined,
+  }],
+  ["/api/v1/nodes/minikube", {
+    apiBase: "/api/v1/nodes",
+    apiPrefix: "/api",
+    apiGroup: "",
+    apiVersion: "v1",
+    apiVersionWithGroup: "v1",
+    resource: "nodes",
+    name: "minikube",
+    namespace: undefined,
+  }],
+  ["/api/foo-bar/nodes/minikube", {
+    apiBase: "/api/foo-bar/nodes",
+    apiPrefix: "/api",
+    apiGroup: "",
+    apiVersion: "foo-bar",
+    apiVersionWithGroup: "foo-bar",
+    resource: "nodes",
+    name: "minikube",
+    namespace: undefined,
+  }],
+  ["/api/v1/namespaces/kube-public", {
+    apiBase: "/api/v1/namespaces",
+    apiPrefix: "/api",
+    apiGroup: "",
+    apiVersion: "v1",
+    apiVersionWithGroup: "v1",
+    resource: "namespaces",
+    name: "kube-public",
+    namespace: undefined,
+  }],
 ];
 
 describe("parseApi unit tests", () => {
-  for (const { url, expected } of tests) {
-    test(`testing "${url}"`, () => {
-      expect(parseKubeApi(url)).toStrictEqual(expected);
-    });
-  }
+  it.each(tests)("testing %s", (url, expected) => {
+    expect(parseKubeApi(url)).toStrictEqual(expected);
+  });
```

Comment:
This just makes use of the jest table testing format, these specific tests weren't causing the problems.